data_steward/tools/load_submission.py

```diff
@@ -0,0 +1,157 @@
+"""
```

Comment:
This is likely my last comment, but I think `load_archived_submission` is a clearer name than `load_submission`.  Someone new looking at the code base may not realize that `load_submission` is pulling from a different source than the `import_rdr_omop` script.